src/CMakeLists.txt

```diff
@@ -72,14 +72,29 @@ target_include_directories(fdbdoc
 target_compile_features(fdbdoc PRIVATE cxx_std_11)
 target_sources(fdbdoc PRIVATE ${ACTOR_G_CPP_FILES})
 
+if(NOT TLS_DISABLED)
```

Comment:
I'd be totally okay if you started the precedent that it should be TLS_ENABLED which is ON by default, and that we should follow the same convention in FoundationDB eventually as well.

Reply:
Yeah that's easier than double negatives LOL.